src/test/scala/org/broadinstitute/dsde/workbench/avram/CommonTestData.scala

```diff
@@ -25,8 +32,15 @@ object CommonTestData {
     slickQueueSize = 1000
   )
 
-  val localDatabase = new SlickDatabaseFactory(localDataSourceConfig).database
+  val configFactory = ConfigFactory.parseResources("app.conf").withFallback(ConfigFactory.load())
+  private val dbcpDataSourceConfig = configFactory.as[DbcpDataSourceConfig]("dbcpDataSource")
+  val dataSource = new DbReference(dbcpDataSourceConfig)
+
+  val collectionName = "collection1"
```

Comment:
This can be fixed later, but I'll still point out that I think these common values aren't adding any value as they currently are... and I'm generally not in favor of shared test data anyway (though sometimes it's just too much work to create a full object graph in the test case). A case where these might be more useful is if they were paired with some correlated data, such as the same values being in a stubbed JSON response.